lib/logstash/inputs/rethinkdb.rb

```diff
@@ -25,6 +25,11 @@ class LogStash::Inputs::RethinkDB < LogStash::Inputs::Base
   # Which databases to watch for changes. Tables added or removed from
```

Comment:
On an unrelated note, `squash` should probably default to `true`. I can open a separate PR for this if you prefer.